content/v2.0/reference/config-options.md

```diff
@@ -38,7 +47,16 @@ influxd \
   --session-length=60 \
   --session-renew-disabled \
   --store=bolt \
-  --tracing-type=log
+  --tracing-type=log \
+  --vault-address=https://localhost:8200 \
+  --vault-ca-cert=/path/to/ca.pem \
+  --vault-ca-path=/path/to/certs/ \
+  --vault-client-cert=/path/to/client_cert.pem \
+  --vault-client-key=/path/to/private_key.pem \
+  --vault-client-max-retries=2 \
+  --vault-client-timeout=60s \
+  --vault-skip-verify \
+  --vault-tls-server-name=secure.example.com
```

Comment:
What is this block of text? Example values? Can we label it?

Reply:
I don't remember exactly why I added this block in the first place. I think it was before I included an example of each flag in the flag descriptions. I'm just going to remove this codeblock.